.travis.yml

```diff
@@ -52,3 +53,5 @@ script:
   - cd docs && make html && cd ..
   # Make sure the library installs.
   - python setup.py install
+  # Run benchmark
+  - python bin/benchmark_pydy_code_gen.py 4 100
```

Comment:
You could make this: `1 10` so it runs faster. Any reason we need this to run with 4 links and 100 time steps?